exonum/src/runtime/mod.rs

```diff
@@ -12,6 +12,69 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Transactions runtime.
+//!
+//! The module containing common building blocks for creating runtimes for the Exonum blockchain.
+//!
+//! Each runtime contains specific services to execute transactions, process events,
+//! and provide user APIs, e.t.c. There is a unified dispatcher that redirects all calls
+//! and requests to the appropriate runtime environment. Thus, users work with it, and not
+//! with a specific runtimes.
+//!
+//! # Service life cycle
+//!
+//! 1. Each runtime has own [artifacts] registry from which users can deploy them. The artifact
+//! identifier is required by the runtime for constructing service instances. In other words,
+//! an artifact identifier means same as class name, and a specific service instance is
+//! the class instance.
+//!
+//! 2. Each validator should request the dispatcher to deploy an artifact and send confirmation
```

Comment:
Maybe use "Each validator node" to distinct validator node itself from validator node administrator?